tests/unit/backend/corpora/dataset_processing/test_process.py

```diff
@@ -104,10 +104,8 @@ def test_extract_metadata(self, mock_read_h5ad):
             numpy.random.randint(10, size=(50001, 5)) * 50, columns=list("ABCDE"), index=(str(i) for i in range(50001))
         )
 
-        tissue = numpy.random.choice([0, 1], size=(50001))
-        assay = numpy.random.choice([0, 1, 2], size=(50001))
-        eth = numpy.random.choice([0, 1], size=(50001))
-        dev = numpy.random.choice([0, 1, 2], size=(50001))
+        eth = tissue = numpy.random.choice([0, 1], size=(50001))
+        assay = dev = sx = numpy.random.choice([0, 1, 2], size=(50001))
```

Comment:
can you use a more informative variable name? I'm assuming sx is for sex?

Reply:
Done

Reply:
Thanks!